### DISCUSSION TITLE
[Random] Previdência privada e investimentos

### DISCUSSION THREAD

**Discussion by guilopes22** [frontendbr/forum#989] [Discussão]
Bom dia galera, tudo bem?

Estou começando como PJ e gostaria de fazer um plano de previdência privada com o objetivo de aposentadoria.

Alguém aqui faz e poderia compartilhar algumas dicas?

Gostaria de saber as melhores opções para investir, se é melhor em bancos como Itaú, Caixa, BB ou outro lugar.

Também gostaria de saber uma média de valor que eu teria que pagar por mês pra "me aposentar" com um salário decente.

**Comment by kivervinicius** [comment]
boa, gostaria de saber também

**Comment by caioincau** [comment]
Eu uso o Adam previdencia Ihttp://adamcapital.com.br/os-fundos/

**Comment by Morette** [comment]
Eu acho que a melhor coisa é fazer investimentos tipo CDB, Tesouro Direto, Fundos de Investimento e até ações, mas claro que isso vai exigir conhecimento e estudo.

Digo isso porque o rendimento é bem maior.

**Comment by cesarlima** [comment]
Concordo com o @Morette , segue alguns canais do youtube com bom conteúdo sobre educação financeira , CDBs, Tesouro Direto e bastante coisa interessante.

https://www.youtube.com/channel/UC8mDF5mWNGE-Kpfcvnn0bUg
https://www.youtube.com/channel/UCaFy7I27Ht8hs6_kmZ8zWuw

**Comment by vicainelli** [comment]
Ótima issue..

O ideal, pelo menos o que eu acredito e tento fazer, é diversificar os investimentos.
Parte em renda fixa, parte em renda variável, nunca depender apenas de um recurso.

**Comment by guilhermejcgois** [comment]
Legal essa sua preocupação @guilopes22! Complementando os canais do YT que o @cesarlima compartilhou, tem um blog legal que pode te ajudar tirando algumas dúvidas sobre bancos, corretoras e produtos financeiros, que [Prof. Gorila](professor.gorila.com.vc), que sou suspeito para falar, mas eles possuem também uma [plataforma](https://gorila.com.vc/) para você poder gerenciar o seu portfólio, manter o _tracking_ da rentabilidade histórica de suas posições, dentre outras coisas.

Imagino que você esteja começando nesse mundo novo, então gostaria de recomendar estes 2 artigos:
- [Melhores corretoras de valores para iniciantes](https://professor.gorila.com.vc/guia-de-corretoras-de-valores/melhores-corretoras/melhores-corretoras-de-valores-para-iniciantes/)
- [Melhores Corretoras de Valores e Distribuidoras para o Pequeno Investidor](https://professor.gorila.com.vc/guia-de-corretoras-de-valores/melhores-corretoras/melhores-corretoras-de-valores-para-o-pequeno-investidor/)

Valeu, forte abraço!

**Comment by caioincau** [comment]
Mas previdência faz parte de diversificar, sendo ótimo para quem investe até 12% da renda nessa modalidade e declara imposto no modo completo.

Foi-se o tempo que previdência era algo ruim, só são ruins essas de bancos, os fundos hj em dia diversificam e jogam de forma mais agressiva, dado que é um dinheiro de muito longo prazo.

**Comment by brunoravanhani** [comment]
Concordo com o que o pessoal falou sobre os investimentos, vale a pena estudar um pouco isso porque realmente rende mais. Alguns bancos para lucrarem, pegam o que você coloca na previdência privada e investem em tesouro direto e ficam com uma partezinha do rendimento.

**Comment by felipefialho** [comment]
Fala @guilopes22, fazia tempo que queria lançar uma discussão assim, então deixei sua issue um pouco mais genérica tá?
_____

Investimentos é um dos meus assuntos favoritos. E de fato, developers em geral (pelo que observo), falam e se preocupam pouco em investir e pensar no futuro.

Estudo investimentos desde meados de 2012 quando fui morar sozinho, e investir e planejar o futuro passou a ser uma necessidade. 

Algumas dicas simples que posso dar, dada a "experiência" adquirida nesse periodo:

- Básico né? Mas não gaste mais do que você ganha
- Faça uma planilha com seus gastos e ganhos financeiros (aplicativos like [Guia Bolso](https://www.guiabolso.com.br/) podem ajudar nisso
- Defina um valor mensal para emergência (guarde em poupança ou renda fixa com resgate diário)
- Defina um valor mensal que você pode investir
- Esses valores (emergência e investimentos) são um compromisso seu com você mesmo, assim que o salário cair, você aplica, sem desculpas
- Defina seu perfil como investidor (Conservador, Moderado, Arrojado)
- Quando mais conservador, mais dinheiro em renda fixa. Quanto mais arrojado, mais dinheiro em renda variável (Criptos entram aqui)
- Tenha planilhas para acompanhar seus investimentos (isso é MUITO importante)
- Boas opções de renda fixa: Tesouro Direto (principalmente atrelados a inflação), CDB, LCI
- Boas opções de renda variável: Ações de boas empresas, Fundos imobiliários, ETFs, Criptomoedas (cuidado total)
- Aplicativos (como [Vérios](https://verios.com.br/), [Magnetis](https://magnetis.com.br/), etc) que usam robôs para escolher os melhores investimentos são sim uma boa alternativa
- Dentro dessas opções defina quais te agradam e qual a % que vai alocar para elas
- Aporte todo mês a % que você estipulou e balanceie para que essa % esteja próxima da planejada
- Se você tomar esse cuidado, a % vai mudar quando determinado investimento valorizar ou desvalorizar, automaticamente você vai ter que comprar o investimento na "baixa" para balancear 
- O caminho é longo e o processo é lento, Cisnes Negros são raros, nada vai te deixar milionário da noite para o dia

😉

**Comment by matheusml** [comment]
Oi @guilopes22 

Vejo muitas pessoas tendo medo da Previdência Pública (com razão) e querendo saber mais sobre a Previdência Privada.

Na minha opinião ambas são terríveis e você deve se afastar das duas.

### Previdência Pública
A Previdência Pública é basicamente um **esquema de pirâmide**. O dinheiro que você bota lá mês a mês não é para a sua aposentadoria, mas para quem já se aposentou. Isso funcionava muito bem no passado, mas atualmente temos dois problemas:

**1. Os idosos vivem cada vez mais**

![](http://www.pastoraldapessoaidosa.org.br/images/stories/fotos2016/6506e1d2.png)

**2. A nossa população, que no passado era de uma maioria de jovens, está ficando cada vez mais velha**

![](http://slideplayer.com.br/slide/352701/2/images/6/Evolu%C3%A7%C3%A3o+et%C3%A1ria+do+Brasil.jpg)

É só juntar as duas peças que a gente percebe que a geração atual não vai ver 1 centavo da aposentadoria do INSS.

### Previdência Privada
A única coisa pior do que a Previdência Pública é a Privada. O Estado na pior das hipóteses pode mudar as regras do jogo, imprimir mais dinheiro, ou fazer qualquer tipo de artimanha pra pagar os aposentados. **Já na Privada, se algo der errado com a instituição contratada já era o seu dinheiro**.

Fora que a Previdência Privada simplesmente vai pegar o seu dinheiro e o investir por você. Mas eles não fazem isso de graça, essa margem de lucro (alta) é o que faz com o que o negócio deles funcione.

![](https://img600.imageshack.us/img600/2448/syep.jpg)

Ou seja, baixa rentabilidade e grande probabilidade de falência.

### Investir
Como as duas modalidades de Previdência são furadas o melhor a fazer é investir você mesmo o seu próprio dinheiro.

Pra quem for CLT como eu, trate o INSS como um imposto. **Se lá no futuro, por algum milagre, você receber um dinheiro na aposentadoria, ótimo. Mas se não receber, você já estava preparado.**

E a Privada, simplesmente nem comece a fazer.

Agora, basta estudar (e muito) como investir o seu próprio dinheiro pra que lá na frente você não dependa de ninguém e possa ter a tão sonhada tranquilidade financeira.

**Comment by andrewsfg** [comment]
Sou PJ atualmente, após bater muita cabeça estou dando os primeiros passos nos investimentos.

Hoje invisto em Tesouro Direto (SELIC e IPCA para uma possível aposentadoria), CDB (maiores que 100% do CDI), COE, fundos de investimentos e debêntures (os de obras do governo, porque tem isenção de IR, porém essa pode ter riscos).

O que vocês acham da Warren? Como ainda não tenho muita noção com renda variável, estou começando a investir lá para ter uma diversificação. Gostaria de saber a opinião de vocês.

**Comment by caioincau** [comment]
Se você já faz tudo isso por fora, só o que o Warren vai te dar, é acesso há dois etfs que você pode comprar em qualquer corretora, não vale a pena @andrewsfg

**Comment by andrewsfg** [comment]
@caioincau, no objetivo que criei com eles, eles investem 34% em ações (eua/br), achei interessante só por isso. Além da possibilidade de sacar a qualquer hora. Porém, até agora não vi muito resultado.

**Comment by ninetails** [comment]
Também recomendo ir direto para Tesouro Direto. Tem corretoras que não cobram pra investir nesta modalidade, no site do Tesouro tem uma lista contendo corretoras e suas respectivas taxas cobradas.

Tem inclusive uma corretora ae que desenvolve utilizando React pra parte de renda fixa e n cobra pra esta modalidade nem pra abrir conta e tem um app voltado pra isso ;x (não, não trabalho/trabalhei lá nem fui pago pra fazer o marketing)

**Comment by caioincau** [comment]
@andrewsfg só o que eles fizeram foram comprar dois ETFs que replicam o índice, como você já montou uma carteira legal de renda fixa, você consegue comprar sozinho essas ETFs e não pagar a fatia da Warren.

**Comment by andrewsfg** [comment]
@ninetails grande Kazuo! Obrigado pelas dicas @caioincau e @ninetails. Vou me interar mais sobre o assunto da ETFS.

Abraços!

**Comment by lucasbesen** [comment]
Vale a pena investir em robôs, como o da [Vérios](https://verios.com.br/), até conseguir uma bagagem de conhecimento sobre investimentos? Ou seria o contrário?

**Comment by kelciocajueiro** [comment]
@lucasbesen eu tinha um post muito bom salvo aqui que comparava alguns robôs de investimentos: Vérios, Magnetis, Warren e Monetus. Infelizmente o autor excluiu o post. Eu acredito que vale a pena sim investir em algum robô desse, desde que você balanceie bem seu dinheiro entre renda fixa e renda variável.

**Comment by felipefialho** [comment]
@lucasbesen Não é má ideia, costumam render bem. Mas na real, quase todas elas fazem um mix entre Tesouro Direto e ETFs, você pode tranquilamente fazer sozinho.

**Comment by lucasbesen** [comment]
@kelciocajueiro poderia falar mais sobre esse balanceamento?

@LFeh você já utilizou algum e recomenda?

**Comment by caioincau** [comment]
Eu usei a Vérios, gostei até começar a estudar e me virar sozinho, mas hoje em dia, recomendaria a Magnetis por ter uma diversificação maior.

Fica a dica da Yubb para procurar onde investir e comprarar esses bots todos.

**Comment by ghost** [comment]
@guilopes22 vou tentar te passar o conhecimento que adquiri trabalhando em corretora:

- Tesouro Direto: Acredito que aqui é o primeiro passo. Você pode até simular algo aqui [Simulador de Renda Fixa](https://easynvest.github.io/simulador-rendafixa) pra ver se vale a pena

- Fundo de Investimento: Aqui tem vários tipos de rendimento e risco, é ótimo pra quem quer dar um segundo passo. Um bom produto pra quem quer diversificar e pensa a longo prazo. Um exemplo de um bom fundo é o [Alaska Black III ](https://docs.wixstatic.com/ugd/8c9dd6_3e67ef59c7d445d09464267623c6ee48.pdf). Percebam que nesse link tem um relatório o quanto o fundo vem perdendo e ganhando. Uma observação, fundo de investimento tem um certo risco e como da pra ver nesse que citei não é algo pra curto prazo.

Eles também tem taxa de administração. Quem quiser saber mais sobre fundos [Sobre Fundos](https://www.youtube.com/watch?v=ztTKCYH046A)

- ETFs: Como muitos já citaram acima é um bom produto, pra quem não sabe você consegue investir em ETFs internacionais é o caso da S&P500 que é um indice com empresas como Amazon, Facebook, Microsoft e muitas outras.

Agora respondendo algumas perguntas originais do tópico:

#### Gostaria de saber as melhores opções para investir, se é melhor em bancos como Itaú, Caixa, BB ou outro lugar.

Corretora é o melhor caminho por vários pontos. Desde os produtos ofertados normalmente com rendimentos melhores que os bancos até sobre as taxas como já citado muitas não cobram em Renda Fixa por exemplo.

#### Também gostaria de saber uma média de valor que eu teria que pagar por mês pra "me aposentar" com um salário decente.

Isso é relativo, se você conseguir 1 milhão investido já da um rendimento suficiente pra você viver dos dividendos se tua carteira for diversificada.

#### O que eu indico:

Procura estudar bastante sobre como diversificar carteira, dá o primeiro passo pequeno mesmo lá no tesouro direto. O importante mesmo é educação e cadência. O restante vai depender do quanto tu estudar sobre o assunto e as decisões que você tomar.

Eu tento diversificar bastante tenho Renda fixa, tenho Fundos de Investimento, tenho ETF`s. E se tu me perguntar o melhor, pra mim depende. Eu tenho investimentos pensando em 3, 5 e 10 anos. E é justamente por ai o caminho que temos que nos educar.

Espero que tenha sido útil pra ti. Caso queira saber mais sobre os assuntos alguns canais de educação que vale a pena conferir:

[Easynvest](https://www.youtube.com/user/easynvest)
[Primo Rico](https://www.youtube.com/channel/UCT4nDeU5pv1XIGySbSK-GgA)
[Me poupe](https://www.youtube.com/channel/UC8mDF5mWNGE-Kpfcvnn0bUg)


Qualquer dúvida só mandar um sinal de fumaça, tweet qualquer coisa =)

**Comment by edumcsouza** [comment]
Oi pessoal,

Sempre tive dúvidas se a previdência vale a pena, ainda mais com a grande quantidade e variedade de investimentos hoje no mercado. Atualmente invisto apenas em ações e fundos, ainda não fui para a previdência. De qualquer forma, quando eu estava decidindo, fiz um [simulador](https://meufuturo.com.vc/simulador/) que calcula quanto você precisa de previdência privada e seguro de vida.

Tem um simulador da própria [Icatu](https://portal.icatuseguros.com.br/previdencia/simulador-minha-aposentadoria) e da [Mongeral](https://simuladores.mongeralaegon.com.br/SimuladorOnline/) também que ajuda bastante.

Um abraço,
